### PR TITLE
Using `constants.yaml` ROS parameters file for road width and Gaussian kernel parameters.

### LINKED ISSUE #84: Use rosparam for road_width in gps_mapper
I can't find how the image is actually being drawn in `interpolate.py`'s `xwindow()` so I don't know how it's defining the road width to be, but I believe this should be a rosparam that we can set (in meters)

### PR DESCRIPTION
**Pull request by diptodip**
Closes #84.

### CODE REVIEW

File: buzzmobile/constants.yaml
Comment (irapha) [inline buzzmobile/constants.yaml:14]: Put this and its comment in a new block. It's not related to image things at all.

**Review by irapha — CHANGES_REQUESTED** [review]

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py]: Don't call `get_param` on every `update_image()`. Save the params as global values on the top of the file.

**Review by irapha — CHANGES_REQUESTED** [review]

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py]: This won't work. Do:

``` python
y_range, x_range, top_left, bottom_right = (
           = interpolate.dimensions(frames['points']))
```

File: buzzmobile/process/gps_mapper/interpolate.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/interpolate.py]: You need a parens here too

File: buzzmobile/process/gps_mapper/interpolate.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/interpolate.py]: parens

File: buzzmobile/process/gps_mapper/interpolate.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/interpolate.py]: parens

**Review by irapha — CHANGES_REQUESTED** [review]

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py]: Should this call to interpolate.dimensions be using `points` as arg instead of `frames['points']`? If the latter isn't set yet, then it will be `None`.

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py:50]: Why do you set `frames['points']` here if it's going to be reset in line 40? Why don't you just set `points` to `[(y, -x) for (x, y) in points]`?

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py]: nit: Can you move the calls to `int(round())` to where you define `height` and `width`?

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (irapha) [inline buzzmobile/process/gps_mapper/gps_mapper.py:20]: Since things seem a bit messy with the ordering of `x, y`, can you add a comment that specifies what `frames['location']` expects? Is it `x, y`, `y, x`, `y, -x`, etc?

**Review by irapha — CHANGES_REQUESTED** [review]
If possible can you add more comments like, everywhere? It's kind of hard to understand what's going on sometimes

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (diptodip) [reply buzzmobile/process/gps_mapper/gps_mapper.py:50]: We calculate the ranges before flipping the points because polylines are weird. This is how I had to handle it when we got it to work last week.

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (diptodip) [reply buzzmobile/process/gps_mapper/gps_mapper.py]: oops yeah replaced accidentally

File: buzzmobile/process/gps_mapper/gps_mapper.py
Comment (diptodip) [reply buzzmobile/process/gps_mapper/gps_mapper.py:20]: T_T ok

**Review by irapha — APPROVED** [review]
Alright. We _definitely_ need to test this before you merge but LGTM.

You have a GPS fix data right? Can you test that it works before merging?

**Comment by diptodip** [comment]
Yeah I can test, but not right now.

**Pull request merged**
